app/controllers/relays_activations_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RelaysActivationsController < ApplicationController
+
+  before_action :authenticate_user!
+  before_action :init_current_facility
+  authorize_resource :facility
```

Comment:
Can you double check this permissioning and possibly add a spec for it? I suspect this will check for `:create`/`:destroy` on `Facility`, which will only be available to global admins. I believe we want to allow facility directors (and probably facility administrators since they pretty much share permissions).

Reply:
I thought I had checked it as a Facility Director locally, but I will double-check!